livemaker/cli/lmpatch.py

```diff
@@ -99,6 +102,9 @@ def lmpatch(archive_file, patched_lsb, split, no_backup, force):
                     sys.exit('{} already exists'.format(p))
 
     lsb_path = PureWindowsPath(patched_lsb)
+
+    path_mode = os.path.isdir(patched_lsb)
+
```

Comment:
So, the -r parameter is required for path patch. 
OK, accepted

Reply:
Yeah, I just think it's better to make the user explicitly state that they are intentionally using a patch directory here, especially since the old behavior was to not support directories at all